config/fcos/v1_6_exp/validate_test.go

```diff
@@ -234,3 +236,110 @@ func TestValidateGrubUser(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateConfig(t *testing.T) {
+	tests := []struct {
+		in      Config
+		out     error
+		errPath path.ContextPath
+	}{
+		// valid config (wipe_table is true)
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Disks: []base.Disk{
+							{
+								Device:    "/dev/vda",
+								WipeTable: util.BoolToPtr(true),
+								Partitions: []base.Partition{
+									{
+										Label: util.StrToPtr("foo"),
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		// valid config (disk is /dev/disk/by-id/coreos-boot-disk)
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Disks: []base.Disk{
+							{
+								Device:    rootDevice,
+								WipeTable: util.BoolToPtr(false),
+								Partitions: []base.Partition{
+									{
+										Label:  util.StrToPtr("bar"),
+										Number: 1,
```

Comment:
We're not actually testing that `rootDevice` is special-cased, since the partition has `Number: 1`.  Let's drop this line.